pkg/os/shell/shell_windows.go

```diff
@@ -0,0 +1,5 @@
+package shell
+
+var (
+	supportedShell = []string{"bash", "powershell", "cmd", "zsh"}
```

Comment:
I would not support bash and zsh on Windows.
This means people run it from Cygwin or WSL?

Reply:
Right in case of Cygwin or WSL side

Reply:
but on WSL you can not communicate with the hypervisor as the command is recognized/using the Linux executable. So this would in that case not work as expected.

Cygwin would work, but I think it is better to use a force option here instead.

Reply:
@gbraad why we need the force option, it does autodetect the shell env and print out as expected, as long as user using it on powershell/cmd it will only give that outputs.

Reply:
I do not think you understand me. Why would you want to support cygwin on Windows, when at times this didn't work as expected? At this point I do not think it is a ood choice.

so, this is why I mention `crc.exe oc-env -s bash` to be an option to FORCE if you insist on trying this on Cygwin's bash.